### DISCUSSION TITLE
Extended keyboard shortcuts for Emacs

### DISCUSSION THREAD

**Discussion by rbutoi** [wezterm/wezterm#808] [Q&A] [answered]
Hello, does anyone have a working config for passing extended keyboard shortcuts through Emacs, including through ssh and tmux? I've tried [`enable_csi_u_key_encoding`](https://wezfurlong.org/wezterm/config/lua/config/enable_csi_u_key_encoding.html) with https://gist.github.com/gnachman/b4fb1e643e7e82a546bc9f86f30360e4 to no avail - many shortcuts didn't work. I can confirm that did work with iTerm2.

**Comment by rbutoi** [comment, marked as answer]
Also posted on reddit and got this helpful answer, answering it here as well: https://old.reddit.com/r/emacs/comments/nfx63q/which_way_to_get_extended_keys_available_in_linux/gyoq649/. All I had to do was have these keys enabled: https://paste.debian.net/1198098/.

**Reply by elken** [reply]
If someone finds this in the future, the Reddit reply has been removed but can still be found via Wayback Machine: https://web.archive.org/web/20210519121110/https://old.reddit.com/r/emacs/comments/nfx63q/which_way_to_get_extended_keys_available_in_linux/gyoq649/
